pjmedia/src/pjmedia/conference.c

```diff
@@ -856,36 +1140,91 @@ static pj_status_t resume_sound( pjmedia_conf *conf )
 /**
  * Destroy conference bridge.
  */
-PJ_DEF(pj_status_t) pjmedia_conf_destroy( pjmedia_conf *conf )
+PJ_DEF(pj_status_t) pjmedia_conf_destroy(pjmedia_conf *conf)
```

Comment:
if possible, refrain from modifying code white spaces. it creates an unnecessarily larger patch that's more difficult to review. I will mark several of these (but not all so it doesn't flood the review) with "*cosmetic" in the comment.